.github/ISSUE_TEMPLATE/detox_nightly_build_failure.md

```diff
@@ -3,7 +3,7 @@ name: Detox Nightly Build Failure Template
 about: Template for reporting a detox nightly build failure
 title: "Bug - Detox - Fix Overnight Failures"
 labels: bug 
-assignees: timwright12
+assignees: SKMOstudio
```

Comment:
@SKMOstudio this is a template for a ticket that gets created with Detox fails overnight. It's usually pretty quiet, but when Detox does fail it will create a ticket and assign it to you to look into. There isn't always something actionable, but if there is, just alert the team that owns the test (usually whoever last updated the file)